### DISCUSSION TITLE
html file from storage returned as text/plain

### DISCUSSION THREAD

**Discussion by rbkayz** [supabase/supabase#7377] [Questions] [answered]
Hi Community,

Need some help. I've stored a html file in storage. However, if I create a signed URL (or even copy the URL from the studio), it opens the in browser as plain markup text

If I inspect, I see that the entire html is wrapped in a pre tag for some reason.

Any way to solve this please?

Also side note, that if I download the file, it opens fine on the browser. 

Thank you

**Comment by GaryAustin1** [comment, marked as answer]
Yes that is by design currently.  Supabase does not want storage used for hosting pages...
https://github.com/supabase/supabase/discussions/2557
